charts/adot-exporter-for-eks-on-ec2/templates/aws-for-fluent-bit/configmap.yaml

```diff
@@ -88,6 +88,8 @@ data:
         log_group_name      /aws/containerinsights/{{ .Values.clusterName }}/application
         log_stream_prefix   ${HOST_NAME}-
         auto_create_group   true
+        {{ if .Values.fluentbit.output.applicationLog.log_retention.enabled }}log_retention_days  {{ .Values.fluentbit.output.applicationLog.log_retention.days }}{{ end }}  
```

Comment:
Wouldn't it make more sense to do something like:
{{ if .Values.fluentbit.output.applicationLog.log_retention.enabled }}
log_retention_days {{ .Values.fluentbit.output.applicationLog.log_retention.days | default 7 }}
{{ end }}

Something along those lines. Otherwise if you don't set log_retention_days but enable... It becomes an issue. Also this would I believe answer the raised points from @Aneurysm9 and @sunnygoel87 

Just started using this Helm chart and was about to raise the same issue so very excited to see this merged in.

Reply:
@jbcom - I agree. Setting the "default" will help address the scenario where user only enabled but didn't set the log_retention_days in values.yaml file. And this should apply to all the logs - Application, dataplane and host

Reply:
@sunnygoel87 @jbcom I agree, by default 7 days of retention for users only enabled but didn't set the log_retention_days.